Exercise-7/CountTheNumberOfWords.java

```diff
@@ -0,0 +1,28 @@
+package exercise7;
+
+import java.util.Scanner;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.StringReader;
+
+public class CountTheNumberOfWords {
+	public static void main(String[] args) throws IOException {
+		Scanner scanner = new Scanner(System.in);
+		String string = scanner.nextLine();
+		int count = 0;
+		try(BufferedReader reader = new BufferedReader(new StringReader(string))){
```

Comment:
Adding descriptive comments explaining the purpose and approach would be helpful while checking the solution at a later point in time.